canopy/design-system/src/styles/layout.scss

```diff
@@ -0,0 +1,119 @@
+.margin-s {
```

Comment:
This should be put into to a function so it is easy to manage changing the values and keep it DRY

Reply:
resolved